src/components/editor/EditorMainIssueController.jsx

```diff
@@ -65,6 +65,12 @@ export default class EditorMainIssueController extends FalcorController {
                 return false;
               }
             }
+            if (/<a.*?href\s*=\s*["'][^http].*?>/.test(article.html)) {
+              if (!window.alert(article.title + " has an <a> tag with a non-absolute link. " +
```

Comment:
The editor's don't necessarily know html, they just input markdown and links into the editor tools, so while there's a possibility they may infer what `<a> tag` means in the context, it's not sure at all, and it's definitely not good UX. It would also be good to not only include the title but also the URL, so instead of doing .test above maybe do .match or .exec or one of the other javascript methods so you can capture the href. 

So if you could just change it to something less jargonish and more clear like `[TITLE] has at least one URL in a non-absolute format (that means it has no http(s):// in front of it) in the body of the article. This will probably cause the link to break in the browser due to something called relative links it will be misinterpreted as. The link was: [URL][NEWLINE] Unless you have discussed overriding this with someone from the development team please cancel the issue from publishing and fix this first. The easiest way to get a correct link is to copy it from your browsers URL bar, remember to prefer https over http.[NEWLINE]Do you want to override our warning and continue publishing?`

I know that's quite long, but especially the first time they see it, we want them to understand it, and not come running to us asking what the error message meant.

I'd love it if you think you can convey the same clarity in less words though, because that one off the top of my head ended up being quite long really.